app/main.py

```diff
@@ -1,13 +1,16 @@
+
+
+
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {"line": error["line_number"], "column": error["column_number"], "message": error["text"], "name": error["code"], "source": "flake8"}
     pass
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
+    return {"errors":[format_linter_error(error) for error in errors], "path": file_path, "status": "failed" if errors else "passed"}
```

Comment:
The 'pass' statement on line 11 is unnecessary because the function already has a return statement. You can remove it.